csvcubed-pmd/tests/behaviour/csvwcommandline.feature

```diff
@@ -57,6 +57,46 @@ Feature: Testing the csvw command group in the CLI
     And the file at "out/junior-roles.csv" should exist
     And the file at "out/gov.uk/schema/junior-roles.json" should exist
 
-    Scenario: The `pull` command should fail when the CSV-W file cannot be found.
+  Scenario: The `pull` command should fail when the CSV-W file cannot be found.
     When the pmdutils command CLI is run with "csvw pull https://example.com/non-existant-file.csv-metadata.json"
-    Then the CLI should fail with status code -1
\ No newline at end of file
+    Then the CLI should fail with status code 1
+
+
+  Scenario: "the 'uri replace' command should get and read in a ttl file but not be able to run without more arguments being entered in the command line"
+    Given the existing test-case file "TurtleTestFile.ttl"
+    When the pmdutils command CLI is run with "uri replace TurtleTestFile.ttl TurtleOutputFile.ttl"
+    Then the CLI should fail with status code 1
+
+  Scenario: "the 'uri replace' command should get and read in a ttl file but not be able to replace single pair of uri"
+    Given the existing test-case file "TurtleTestFile.ttl"
+    When the pmdutils command CLI is run with "uri replace TurtleTestFile.ttl TurtleOutputFile.ttl -v file:/tmp/qb-id-10002.csv http://example.com/some-dataset"
+    Then the CLI should fail with status code 1
+  
+  Scenario: "the 'uri replace' command should get and read in a ttl file and find and replace a single pair uri by being '--force' to continue through 'WARNING'"
+    Given the existing test-case file "TurtleTestFile.ttl"
+    When the pmdutils command CLI is run with "uri replace TurtleTestFile.ttl TurtleOutputFile.ttl -v file:/tmp/qb-id-10002.csv http://example.com/some-dataset --force"
+    Then the CLI should succeed
+
+  Scenario: "the 'uri replace' command should get and read in a ttl file but not be able to replace multiple pairs of uri"
+    Given the existing test-case file "TurtleTestFile.ttl"
+    When the pmdutils command CLI is run with "uri replace TurtleTestFile.ttl TurtleOutputFile.ttl -v file:/tmp/qb-id-10002.csv http://example.com/some-dataset -v file:/tmp/d-code-list.csv http://example.com/some-dataset/d-code-list"
+    Then the CLI should fail with status code 1
+
+  Scenario: "the 'uri replace' command should get and read in a ttl file and find and replace multiple pairs uri by being '--force' to continue through 'WARNING'"
```

Comment:
Again, I'm a bit confused about what these scenarios are testing. Maybe we can have a think about some better names.